Tools/CISupport/ews-build/steps.py

```diff
@@ -5323,6 +5324,7 @@ def run(self):
                            '--json-output={0}'.format(self.jsonFileName)]
         else:
             self.command = self.command + customBuildFlag(platform, self.getProperty('fullPlatform'))
+        self.command = self.shell_command(' '.join(self.command) + ' > logs.txt 2>&1 ; grep Ran logs.txt')
```

Comment:
grepping for Ran also includes words like Random, can grep for "Ran " (Ran with a space)